src/services.py

```
@@ -139,3 +139,28 @@ def create_service(name: str, description: str = "", type: str = "service", url:
     if url:
         body["url"] = url
     return api_caller.call_post("/services", body=body)
+
+@service_mcp.tool(annotations={"readOnlyHint": False, "title": "Update Service"})
+def update_service(service_id: str, name: str, description: str = "", type: str = "service", url: str = None, tier: int = 3):
```

Ruff (0.15.7)

[error] Function argument `type` is shadowing a Python builtin

(A002)